pom.xml

```diff
@@ -3053,6 +3053,17 @@
         <version>${jackson1.version}</version>
       </dependency>
       <!-- jackson 2 (used by addon) -->
+      <dependency>
+        <groupId>org.codehaus.woodstox</groupId>
+        <artifactId>stax2-api</artifactId>
+        <version>4.2</version>
+        <exclusions>
+          <exclusion>
+            <artifactId>stax-api</artifactId>
+            <groupId>javax.xml.stream</groupId>
+          </exclusion>
+        </exclusions>
+      </dependency>
```

Comment:
Could you move this elsewhere that the Jackson section please, probably next to the `stax-api` dependencies.

Reply:
Done ! Now near stax-api